docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+   
+  * METplus methods: S2S Use Cases.
+    
+What is the standard for comparison that provides a reference level of skill (e.g., persistence, climatology, reference model)?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Climatologies or Reference models may be passed into METplus using the
+following configuration options:
+
+* {MET TOOL}_CLIMO_MEAN
+  
+* {MET TOOL}_CLIMO_STDEV
+   
+* {MET TOOL}_CLIMO_CDF
+   
+This can be found in Grid-Stat, Point-Stat, Gen-Ens-Prod, and Ensemble-Stat
+tools.
+
+What is the geographic location of the model data being evaluated? Are there specific areas of interest for the evaluation?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Masking regions are what METplus uses to define verification areas of
+interest. These can be defined prior to running tools using the
+Gen-Vx-Mask tool, or during run-time using the METPLUS_MASK_DICT options.
+
+What domain should be used to evaluate on the model domain, observation domain (if gridded), or some other domain?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The decision to evaluate on model or observation/analysis domain is
+user-specific but the user may want to consider the following:
+
+* Regridding to the courser domain will smooth high resolution information
+  that may be important but smoother forecasts tend to score better.
+   
+* Regridding to a finer domain essentially adds in additional information
+  that is not real.
+   
+* One way to avoid the interpolation debate is to regrid both to a third
+  grid.
+   
+Regridding in METplus can be completed using the Regrid-Data-Plane tool if
+the fields will be used more than once.
+
+Regridding can also be done on the fly using the {Tool}_REGRID_TO_GRID.
+All grid-to-grid verification tools have the regridding capability in it.
+
+What is the evaluation time period? Retrospective with specific dates? Ongoing, near-real-time evaluation? Or both retrospective and realtime?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Basically, running retrospectively means that the observations/analyses are
+already available on disk and running in realtime is when the system needs
+to wait for the observations to be available on the system.
+
+In METplus, the LOOP_BY configuration can be used.
+
+LOOP_BY = VALID or REALTIME to have METplus proceed through the data based
+on Valid Time.
+
+LOOP_BY = INIT or RETRO to have METplus proceed through the data based
+on Initialization Time.
+
+How should the testing and evaluation project be broken down into METplus
+Use Cases? One large one or multiple smaller ones?
+
+* How will METplus be run? Manually? Scheduled, through cron?
```

Comment:
This seems like it should be its own header, like "What is the evaluation time period?" above.

Reply:
@j-opatz I need some help with this.  I've given the questions on line 242 and 245 their own sections.  Unfortunately, there is no answer for the questions starting on line 242. 
I'll do my best to push the file to github in the next hour.

Reply:
See sections 2.1.1.7 and 2.1.1.8
https://metplus.readthedocs.io/en/feature_650_questions2ask/Users_Guide/getting_started.html